### PR TITLE
WeirdcoreHud

### PR DESCRIPTION
**Pull request by zTomma**

### CODE REVIEW

File: hud-data/weirdcorehud.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{
+  "name": "WeirdcoreHUD",
+  "author": "Tomma",
+  "contributors": [
+    "Yara"
+  ],
+  "releaseDate": "2024 04 06",
```
Comment (rbjaxter) [inline hud-data/weirdcorehud.json:7]: Format as `2024-04-06`

File: hud-data/weirdcorehud.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  ],
+  "tags": [
+    "vaporwave"
+  ],
+  "repo": "https://github.com/zTomma/WeirdcoreHUD",
+  "hash": "7516de9cbdf71cfc0c07268651f083e7d2a15d88",
+  "prerelease": false,
+  "social": {
+    "steam_profile": "76561199089787762",
+    "twitter": "@zTomma_",
```
Comment (rbjaxter) [inline hud-data/weirdcorehud.json:28]: May work either way but I believe most huds here list their twitter without the `@` (is this something that the checks would pick up @mastercoms?)

File: hud-data/weirdcorehud.json [hunk 3]
```diff
@@ -0,0 +1,68 @@
+{
+  "name": "WeirdcoreHUD",
+  "author": "Tomma",
+  "contributors": [
+    "Yara"
+  ],
+  "releaseDate": "2024 04 06",
+  "resources": [
+    "https://youtu.be/hlAn1QJPX7o"
```
Comment (rbjaxter) [inline hud-data/weirdcorehud.json:9]: To your resources, you'd also want to add the image you included (`sfondo-hud`). You'd probably want to list it before the youtube link as well

**Review by rbjaxter — CHANGES_REQUESTED** [review]

File: hud-data/weirdcorehud.json (hunk 3 above)
Comment (zTomma) [reply hud-data/weirdcorehud.json:9]: fixed, thx  for warning me

**Pull request merged**